package.json

```diff
@@ -5,7 +5,7 @@
   "homepage": "https://iterable.com/",
   "repository": {
     "type": "git",
-    "directory": "https://github.com/Iterable/iterable-web-sdk.git"
+    "url": "https://github.com/Iterable/iterable-web-sdk.git"
```

Comment:
unrelated typo - this will make sure the NPM package has a link back to the GitHub repo